<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVStereoCalibration</class>
 <widget class="QDialog" name="OpenCVStereoCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>419</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OpenCV Stereo Calibration Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Flags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="fixed_principal_point">
        <property name="toolTip">
         <string>The principal point is not changed during the global optimization, it stays at the center</string>
        </property>
        <property name="text">
         <string>fixed principal point</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fixed_aspect_ratio">
        <property name="toolTip">
         <string>The function considers only fy as a free parameter, the ratio fx/fy is computed and used further</string>
        </property>
        <property name="text">
         <string>fixed aspect ratio</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="same_focal_length">
        <property name="toolTip">
         <string>Enforces fx1 = fx2 and fy1 = fy2</string>
        </property>
        <property name="text">
         <string>same focal length</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tangential_distortion">
        <property name="toolTip">
         <string>Set and keep tangential distortion coefficients (p1, p2) to zero?</string>
        </property>
        <property name="text">
         <string>tangential distortion</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fix_intrinsic">
        <property name="toolTip">
         <string>If it is set, &lt;b&gt;cameraMatrix&lt;/b&gt; , as well as &lt;b&gt;distCoeffs&lt;/b&gt; are fixed, so that only R, T, E and F are estimated</string>
        </property>
        <property name="text">
         <string>fix intrinsic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="intrinsic_guess">
        <property name="toolTip">
         <string>cameraMatrix contains valid initial values of
fx, fy, cx, cy that are optimized further.
</string>
        </property>
        <property name="text">
         <string>use intrinsic guess</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Extrinsic Matrices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="relative_to_cam1">
        <property name="text">
         <string>relative to first camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="relative_to_pattern">
        <property name="text">
         <string>relative to calibration pattern</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Minimum correspondences</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="minCorrespondences">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpenCVStereoCalibration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenCVStereoCalibration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
